mappings/net/minecraft/client/render/AtlasTextureId.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4730 net/minecraft/client/render/AtlasTextureId
```

Comment:
Just a note: Mojang seems to call this `Material` based on the toString.

Reply:
however, from usage, it is more of an id for a sprite (as seen in textured render layers class)

Reply:
AtlasTextureId is better than material. However, name should probably also refer to Sprite. The atlas texture includes all sprites, and this identifies a specific sprite within a specific atlas texture.

Reply:
Renamed it to `SpriteIdentifer` given `Sprite` is always in an atlas in minecraft.